src/styles/main.css

```diff
@@ -1,3 +1,9 @@
+/* remove default margin/padding */
+* {
```

Comment:
Are you just trying to remove the margin/padding around the window? I'd change this from `*` to `body` so it doesn't affect any margins and paddings elsewhere.